extensions.yaml

```diff
@@ -43,10 +44,27 @@ extensions:
       - kernel-rt-modules
       - kernel-rt-modules-extra
       - kernel-rt-devel
-  # https://github.com/openshift/machine-config-operator/pull/2376/
+  # https://github.com/openshift/machine-config-operator/pull/2456
+  # https://github.com/openshift/enhancements/blob/master/enhancements/sandboxed-containers/sandboxed-containers-tech-preview.md
   # GRPA-3123
-  qemu-kiwi:
+  # The dependency tree here is:
+  # - kata-containers (rhocp)
+  #   - qemu-kiwi (advanced-virt)
+  #     - ipxe-roms-qemu (appstream)
+  #     - libpmem (appstream)
+  #     - pixman (appstream)
+  #     - seabios-bin (advanced-virt)
+  #     - seavgabios-bin (advanced-virt)
+  #     - sgabios-bin (advanced-virt)
+  sandboxed-containers:
     architectures:
       - x86_64
     packages:
+      - kata-containers
       - qemu-kiwi
+      - ipxe-roms-qemu
+      - libpmem
+      - pixman
+      - seabios-bin
+      - seavgabios-bin
+      - sgabios-bin
```

Comment:
As long as `qemu-kiwi` has its dependencies properly enumerated in the RPM, `rpm-ostree` will do the necessary depsolving to pull them in.

You could probably even reduce the extension to just specify `kata-containers` and it should pull in `qemu-kiwi` and then all the deps

Reply:
Perfect!